generated/api.github.com.json

```diff
@@ -28033,6 +28033,7 @@
           { "$ref": "#/components/parameters/issue-number" }
         ],
         "requestBody": {
+          "required": true,
```

Comment:
This is my fix - implemented in this repo itself - to make the "Remove assignees" API reflect reality. We can release it as a `fix`.